pkg/services/alerting/notifiers/base.go

```diff
@@ -10,15 +10,15 @@ import (
 )
 
 const (
-	triggMetrString = "Triggered metrics:\n\n"
+	triggeredMetricsString = "Triggered metrics:\n\n"
```

Comment:
This appears to be used internally.

Normally, grafana/go tokens are are fairly verbose, there didn't seem to be any justification for the truncation here.